targets/targets.json

```diff
@@ -21,6 +21,10 @@
                 "help": "Default network interface type. Typical options: null, ETHERNET, WIFI, CELLULAR, MESH",
                 "value": null
             },
+            "deep-sleep-latency": {
+                "help": "Time in ms required to go to and wake up from deep sleep (max 10)",
```

Comment:
I assume `MBED_CONF_TARGET_DEEP_SLEEP_LATENCY` is the worst case scenario (the main depending factor here are the clocks that are running and need to be reenabled - can change in the runtime - in the most cases now it wont).

Max number is defined in the help text as 10, but not checked anywhere?

Reply:
I understood, just checking if we set it to 11 , is this allowed and just impacts the system (no compile time error). I believe so.